cypress/integration/components/navbar/navbar.spec.js

```diff
@@ -13,28 +14,28 @@ describe("Profile Page | CodeLabz", () => {
   it("check only mini navbar is showing to non loggedin user", function () {
     cy.visit(this.base_url);
     cy.wait(1000);
-    cy.get("[data-testId=navbarNonloggedIn]").should("exist");
-    cy.get("[data-testId=navbarloggedIn]").should("not.exist");
+    cy.get("[data-testid=navbarNonloggedIn]").should("exist");
```

Comment:
is this `[data-testid=navbarNonloggedIn]` id used in other tests as well?

Reply:
No, I think the ```[data-testid=navbarNonloggedIn]``` is used anywhere else for testing

its just that there were warnings on console while using ```data-testId``` than ```data-testid```